mythril/analysis/module/modules/exceptions.py

```diff
@@ -25,7 +25,7 @@
 class LastJumpAnnotation(StateAnnotation):
     """State Annotation used if an overflow is both possible and used in the annotated path"""
 
-    def __init__(self, last_jump: int = None) -> None:
+    def __init__(self, last_jump: int = 0) -> None:
```

Comment:
Changing the type-hint from `int` to `Optional[int]` is more precise than changing the default value.

Reply:
True, `Optional[int]` is a good patch.